src/main/java/com/cegeka/academy/service/mapper/ChallengeMapper.java

```diff
@@ -46,5 +50,23 @@ public static ChallengeCategory convertChallengeCategoryDTOToChallengeCategory(C
 
     }
 
+    public static ChallengeCategoryDTO convertChallengeCategoryToChallengeCategoryDTO(ChallengeCategory challengeCategory){
+
+        if(challengeCategory == null){
+
+            return null;
```

Comment:
Ma gandesc daca nu cumva in loc sa returnam "null" ar fi mai bine sa aruncam o exceptie. Ce parere ai?